src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationConstants.cs

```diff
@@ -13,9 +13,19 @@ public static class LinkedInAuthenticationConstants
 {
     public static class Claims
     {
-        public const string PictureUrl = "urn:linkedin:pictureurl";
+        public const string Picture = "picture";
```

Comment:
Renaming public/protected members is a breaking change. If they are no longer used/needed, they should be marked as obsolete instead.